### PR TITLE
fix #6792: link to gc.com checker

### LINKED ISSUE #6792: Support gc.com solution checker
gc.com recently introduced an own solution checker / coordinate checker as an alternative to external geochecker solutions.

At the moment c:geo just ignores the presence of this checker and does not display it.
c:geo should support the checker in some way.

- It could be done in a combined approach also enabling #6782 (handing coords over to external geochecker).
- Implementation wise it could either be done using a dedicated menu with coordinate input or implement it somehow into the existing waypoint menu (Add waypoint and have a possibility to use the WP coords as input for the checker).

### PR DESCRIPTION
**Pull request by Bananeweizen**

### CODE REVIEW

File: main/src/cgeo/geocaching/connector/gc/GCConstants.java [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 import android.support.annotation.NonNull;
 
+import org.apache.commons.lang3.StringUtils;
+
 import java.util.Locale;
 import java.util.regex.Pattern;
 
-
-import org.apache.commons.lang3.StringUtils;
+import cgeo.geocaching.log.LogType;
```
Comment (UniQP) [inline main/src/cgeo/geocaching/connector/gc/GCConstants.java]: According to our coding conventions, the old import order was correct.

File: main/src/cgeo/geocaching/utils/CheckerUtils.java [hunk 2]
```diff
@@ -37,6 +39,10 @@ public static String getCheckerUrl(@NonNull final Geocache cache) {
                 }
             }
         }
+        // GCs own checker
```
Comment (UniQP) [inline main/src/cgeo/geocaching/utils/CheckerUtils.java]: "GCs" → "GC's"

**Comment by Lineflyer** [comment]
retest this please

**Comment by Lineflyer** [comment]
I don't know if @Bananeweizen has time to tackle the requested changes on this one, otherwise @UniQP we could merge and you put the changes on top?

**Pull request merged**